packages/apollo-datasource-rest/src/__tests__/RESTDataSource.test.ts

```diff
@@ -31,13 +31,13 @@ describe('RESTDataSource', () => {
 
   describe('constructing requests', () => {
     it('interprets paths relative to the base URL', async () => {
-      const dataSource = new (class extends RESTDataSource {
+      const dataSource = new class extends RESTDataSource {
```

Comment:
I'm not really sure how/why my PR failed to pass the linter check for this file after updating from `master` if the other updates to `master` were also merged in but somehow didn't fail this check?

As you can see, [I didn't modify these lines at all](https://github.com/apollographql/apollo-server/blob/867a94f2a6eb65243423b6bfe43aff61f1f25df7/packages/apollo-datasource-rest/src/__tests__/RESTDataSource.test.ts#L34), yet somehow my linter checks were failing after #2572 was merged in despite that passing.